zokrates_stdlib/tests/tests/snark/gm17.zok

```diff
@@ -26,34 +26,27 @@
 // 5. Generate the test case
 // 
 // ```sh
-// rm gm17.json
-// touch gm17.json
-
-// echo '{
-// 	"entry_point": "./tests/tests/snark/gm17.zok",
-// 	"curves": ["Bw6_761"],
-// 	"tests": [
-// 		{
-// 			"input": {
```

Comment:
`abi` flag was missing

Reply:
nice, feel free to merge if you're happy with it